Tests/conf.json

```diff
@@ -3369,7 +3369,7 @@
         "_comment2": "~~~ UNSTABLE ~~~",
         "Tenable.sc": "unstable instance",
         "ThreatConnect v2": "unstable instance",
-
+        "Infoblox": "Unstable instance",
```

Comment:
Let's create an issue to keep discussing the instance issue and mention it here.